common/culture/traditions/00_combat_traditions.txt

```diff
@@ -2151,21 +2223,37 @@ tradition_longbow_competitions = {
 	
 	is_shown = {
 		OR = {
+			# Warcraft
 			like_heritage_west_germanic_trigger = yes
 			like_heritage_brythonic_trigger = yes
 			like_heritage_goidelic_trigger = yes
+			AND = {
+				bow_culture_trigger = yes 
```

Comment:
You shouldn't use vanilla traditions when there should be new traditions.

One tradition for night elves that unlock their MaAs and buff archers and another one for hight elves that unlock their MaAs and buff archers.

Unlocking vanilla MaA tradition without MaAs is no use.

Reply:
We should carry this issue over to a separate PR where we focus on adapting the maa innovations to traditions

Reply:
Split off in #996